### ISSUE TITLE
About injections…

### ISSUE THREAD

**Issue by jpommerening** [LaxarJS/laxar-vue-adapter#7]
https://vuejs.org/v2/api/#provide-inject

I think we should have a look at this?

**Comment by alex3683** [comment]
Hm, looks a bit like this was made for us 😃

**Comment by jpommerening** [comment]
Yeah, I think we could just let the user specify the `inject` property and pass it to the `Component` function without even looking at it.
We can then pass `provide` as an option when instantiating the component, maybe even the raw widget services object with all its getters.

React has something similar called `context` but the [documentation](https://facebook.github.io/react/docs/context.html) is littered with warnings not to use it.
